.travis.yml

```diff
@@ -1,5 +1,4 @@
 sudo: false
-cache: bundler
```

Comment:
Done.  That fixed the 2 new failures.  The remaining 2 are already broken on master.